src/services/get-aggregate-service.js

```diff
@@ -15,6 +16,16 @@ export async function AggregateSearchService(searchParams, gridType) {
   if (JWT && isSTACTokenAuthEnabled) {
     requestHeaders.append('Authorization', `Bearer ${JWT}`)
   }
+  if (store.getState().mainSlice.appConfig.STAC_HEADER_COOKIES.length > 0) {
```

Comment:
Might be nice to not require the `STAC_HEADER_COOKIES` config value to exist in the `config.json` if you don't plan on using it (rather than it having to exist and be an empty list). This change would be necessary in multiple locations, at the moment.

```suggestion
  if ((store.getState().mainSlice.appConfig.STAC_HEADER_COOKIES ?? []).length > 0) {
```